beta/src/content/apis/react-dom/client/createRoot.md

```diff
@@ -7,7 +7,7 @@ title: createRoot
 `createRoot` lets you create a root to display React components inside a browser DOM node.
 
 ```js
-const root = createRoot(domNode, options?)
+const root = createRoot(domNode, options?);
```

Comment:
I think we intentionally don't use semicolons on these intro lines because they're more like definitions and aren't sypposed to be valid JS

Reply:
oh, got it. thanks for your explanation.